cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlApiImpl.java

```diff
@@ -94,8 +96,8 @@ private static HTTPHeader generateAuthHttpHeader(String user, String password) {
 
     protected static HTTPHeader getAuthHttpHeader(boolean apiAuthEnabled, Secret apiSecret) {
         if (apiAuthEnabled) {
-            String password = new String(Util.decodeFromSecret(apiSecret, CruiseControl.API_ADMIN_PASSWORD_KEY), StandardCharsets.US_ASCII);
-            return generateAuthHttpHeader(CruiseControl.API_ADMIN_NAME, password);
+            String password = new String(Util.decodeFromSecret(apiSecret, API_ADMIN_PASSWORD_KEY), StandardCharsets.US_ASCII);
+            return generateAuthHttpHeader(API_ADMIN_NAME, password);
```

Comment:
Is there any specific reason we change to use a static import here?

Reply:
No, it's just that I prefer them in most, but not all, cases. That said, this change was unneeded, so I reverted back.